content/yaml-basic-configuration/environment-variables.md

```diff
@@ -56,6 +56,8 @@ Here is a list of the built-in environment variables with brief explanations:
 | CM_ARTIFACT_LINKS       | Information about generated build artifacts that is available in post-publishing step. Read more about it below.                                                |
 | CM_WORKFLOW_NAME        | Name of the workflow that is currently being built       
 | CM_ENV                  | Environment file that sets/overrides environment variables at build time
+| CM_TRIGGER_SOURCE       | The source that triggered the build. Values are `webhook`, `schedule`, or `api`
+| CM_BUILD_STARTED_BY     | The email of the user who started the build if `CM_TRIGGER_SOURCE = api`
```

Comment:
Maybe it's better to say "...the user who started the build from the UI".

Reply:
technically not just from UI, api call counts as well.
Since user is using the account's personal token.

Reply:
My point was that users must not know what the other environment variable holds. So sth like "The email address of the user that triggered the build. Available for builds started from the UI or via the REST API."

Reply:
I'd probably rephrase it to start with when it's defined.
```
Used only with builds triggered by `api` (see `CM_TRIGGERED_SOURCE`). The email of the user who started the build.
```